database/migrations/default/1744826895218_ems_remove_match_status_constraint/up.sql

```diff
@@ -0,0 +1,47 @@
+alter table ems__incidents
+drop constraint matched_crash_pks_non_null,
+add column created_by text not null default 'system',
+add column updated_by text not null default 'system';
+
+comment on column ems__incidents.matched_crash_pks is 'The IDs of multiple crashes that were found to match this record. Set via trigger.''';
+
+--
+-- Trigger function that sets the ems__incident `crash_pk` based on the
+-- provided person_id and updates the crash_pk of all EMS records that
+-- share the same incident number
+--
+create or replace function update_person_crash_ems_match()
+returns trigger as $$
+DECLARE
+    matching_person_record RECORD;
+BEGIN
+    IF NEW.person_id is null
+        THEN return null;
+    END IF;
+    -- Find matching person record
+    SELECT id, crash_pk INTO matching_person_record 
+    FROM people_list_view 
+    WHERE id = NEW.person_id;
+    
+    -- Update crah_pk of related EMS if not already matched to a crash
```

Comment:
little typo here crah_pk

also - so this will allow users to assign different ems records of the same incident to different crash_pks. which i think is okay right?

i was playing around with adding in random person ids to the ems records so that the records would be assigned different crash ids, and it does seem like a needed feature to be able to null out crash id bc now when i null out the person ids i added that pulled up random crashes, i am no longer getting the recommended list of people from the automated crash matches, i am getting lists of people from the random ids i put in. so esp if someone makes a typo typing something in and it changes the crash id and they cant undo that, that would be a problem

Reply:
> so this will allow users to assign different ems records of the same incident to different crash_pks. which i think is okay right?

Yes, I think that's ok. We could have a situation where multiple EMS records are assigned to the wrong crash/person, and the user would have to re-assign each one individually. So we'd have a state during the QA/QC process where EMS records of the same incident are temporarily associated with a different crash.